setup.sh

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env bash
 
-rm -rf cedict makemeahanzi frontend/tagmanager
-
 git clone https://github.com/skishore/makemeahanzi.git makemeahanzi
```

Comment:
Lets add some checks that verify the folders don't exist. If such a folder already exists, let's not download the stuff again.